package.json

```diff
@@ -56,7 +56,6 @@
     "gulp-git": "1.7.2",
     "gulp-help": "1.6.1",
     "gulp-if": "2.0.0",
-    "gulp-image-diff": "0.3.0",
```

Comment:
had to remove this since it had compatibility issues. i dont think we use it right now anyways